modules/entity/spec/sorted_state_adapter.spec.ts

```diff
@@ -89,7 +90,7 @@ describe('Sorted State Adapter', () => {
   it('should remove existing and add new ones on addAll (deprecated)', () => {
     const withOneEntity = adapter.addOne(TheGreatGatsby, state);
 
-    const withAll = adapter.setAll(
+    const withAll = adapter.addAll(
```

Comment:
wrong rebase? `addAll` is deprecated in favor of `setAll` (this also applies for the rest of the tests).

Reply:
look at the test title. It was supposed to test addAll() not setAll(). There is already another test on setAll()

Reply:
Ah, I see. Yeah, it's not necessary to test them both, just use `setAll`. The `setAll` is tested separately to make sure it's the same as `addAll`.